pkg/webhook/loadbalancer/validator.go

```diff
@@ -79,20 +58,6 @@ func (v *validator) Resource() admission.Resource {
 	}
 }
 
-func (v *validator) matchAtLeastOneVmi(lb *lbv1.LoadBalancer) (bool, error) {
-	selector, err := utils.NewSelector(lb.Spec.BackendServerSelector)
-	if err != nil {
-		return false, err
-	}
-
-	vmis, err := v.vmiCache.List(lb.Namespace, selector)
-	if err != nil {
-		return false, err
-	}
-
-	return len(vmis) > 0, nil
-}
-
 func checkListeners(lb *lbv1.LoadBalancer) error {
 	nameMap, portMap, backendMap := map[string]bool{}, map[int32]int{}, map[int32]int{}
 	for i, listener := range lb.Spec.Listeners {
```

Comment:
User may input empty `Listeners`. Do we want to add a check here?

Reply:
Reasonable to check empty Listeners.